sql_export/wizard/wizard_file.py

```diff
@@ -70,12 +70,10 @@ def export_sql(self):
                 else:
                     variable_dict[field.name] = self[field.name]
         if "%(company_id)s" in sql_export.query:
-            company_id = self.env.context.get(
-                "force_company", self.env.user.company_id.id
-            )
+            company_id = self.env.company.id
```

Comment:
I suppose that the force_user/ force_company was to be able to create dedicated views according action menu context ie: one dedicated view per company (instead of the current company) can be confused at some point, not sure what's the best todo !

Reply:
No, these are kind of special parameters, you can use in the sql query %(company_id)s so it will be replaced by the company of the user.
Same for the user_id

It may be usefull to have a dynamic query, that depends on the user and/or of the company of the user, that execute it.
So, the same query won't give the same csv file depending on which user execute it.

The force_company context is deprecated now in favor of the current company on the environment

But indeed, the sql export menu does not manage multi company for now, we may add a company_id field on it and add some ir.rules, in order to display only the queries made for the allowed companies of the user, it would be a nice improvement for intercompany projects